### PR TITLE
chore(issue-checker): 对comment中的未知链接进行提醒

### PR DESCRIPTION
**Pull request by IzakyL**

### CODE REVIEW

File: .github/issue-checker.yml [hunk 1]
```diff
@@ -558,3 +558,8 @@ comments:
     mode: add
     skip-if:
       - "Log upload failed"
+  - name: Check Link
+    regexes: 'https?://(?!.*(github\.com|maa\.plus|prts\.plus))[^\s]+'
```
Comment (Constrat) [inline .github/issue-checker.yml:562]: This is very restrictive.
Users usually upload videos in issues, or images from other domains. I don't think whitemasking links like this is the ideal solution

Comment (IzakyL) [reply .github/issue-checker.yml:562]: The issue-checker did not hide the links, but just add a warn.
Users can judge the links by themselves. If they saw websites like youtube, etc, they can also interview them though there is a warn.

Comment (SherkeyXD) [reply .github/issue-checker.yml:562]: > This is very restrictive.
> Users usually upload videos in issues, or images from other domains. I don't think whitemasking links like this is the ideal solution

This is due to some recent attacks from comments where hackers try to mislead users to download malware from links, like these

Comment (SherkeyXD) [reply .github/issue-checker.yml:562]: > > This is very restrictive.
> > Users usually upload videos in issues, or images from other domains. I don't think whitemasking links like this is the ideal solution


This is due to some recent attacks from comments where hackers try to mislead users to download malware from links, like these

**Comment by Constrat** [comment]
@SherkeyXD Seems pretty dangerous for the community, in that case I have no issue.

**Comment by Rbqwow** [comment]
咳咳咳嗯 我来指手画脚了(跑
github的换行这里一个回车就够了 两个太宽啦

```
---
⚠️当心，此评论可能包含未经验证的链接。⚠️
⚠️Caution, this comment may contain unverified links.⚠️
```

---
⚠️当心，此评论可能包含未经验证的链接。⚠️
⚠️Caution, this comment may contain unverified links.⚠️

**Review by ABA2396 — DISMISSED** [review]

File: .github/issue-checker.yml (hunk 1 above)
Comment (ABA2396) [reply .github/issue-checker.yml:562]: `(github\.com|maa\.plus|prts\.plus)` 是否需要 `\b` 包起来？现在这样的话如果里面带了 `maa.plusss.xx` 之类的也可以吧

**Review by ABA2396 — COMMENTED** [review]
`(github\.com|maa\.plus|prts\.plus)` 是否需要 `\b` 包起来？现在这样的话如果里面带了 `maa.plusss.xx` 之类的也可以吧

File: .github/issue-checker.yml (hunk 1 above)
Comment (status102) [inline .github/issue-checker.yml:562]: `https?:\/\/(?!([^\/]*?\.)?(github\.com|maa\.plus|prts\.plus)\/?)[^\s]+`
放一个正则在这，顺眼就改

**Comment by ABA2396** [comment]
https://www.jyshare.com/front-end/7625/#!flags=&re=%5E(%3F!https%3F%3A%2F%2F(%5B%5E%2F%5D%2B%5C.)%3F(maa%5C.plus%7Cprts%5C.plus%7Cgithub%5C.com)(%2F%7C%24)).%2B%24
`^(?!https?://([^/]+\.)?(maa\.plus|prts\.plus|github\.com)(/|$)).+$`
image
试试这个

**Comment by Constrat** [comment]
> https://www.jyshare.com/front-end/7625/#!flags=&re=%5E(%3F!https%3F%3A%2F%2F(%5B%5E%2F%5D%2B%5C.)%3F(maa%5C.plus%7Cprts%5C.plus%7Cgithub%5C.com)(%2F%7C%24)).%2B%24


This is cool

**Comment by ABA2396** [comment]
> 你们看看这个？`(://|^)([^\.\s/]+?\.)*((?!maa\.plus|github\.com)[^\.\s/]+?\.[^\.\s/]+?)(/|$)`
> 
> [regex101.com/r/QA98tN/1](https://regex101.com/r/QA98tN/1)
> 
> [Visualize](https://regex-vis.com/?r=%28%3A%2F%2F%7C%5E%29%28%5B%5E%5C.%5Cs%2F%5D%2B%3F%5C.%29*%28%28%3F%21maa%5C.plus%7Cgithub%5C.com%29%5B%5E%5C.%5Cs%2F%5D%2B%3F%5C.%5B%5E%5C.%5Cs%2F%5D%2B%3F%29%28%2F%7C%24%29)

https://maa.pluss/xy.zip
虽然没有pluss这个域名）

**Comment by Constrat** [comment]
How is that invalid lol

**Comment by Rbqwow** [comment]
there's a space before chore:
lmao

**Comment by SherkeyXD** [comment]
> How is that invalid lol

The commit message actually starts with ` chore` instead of  `cchore XD

**Comment by Constrat** [comment]
image
😆

**Comment by ABA2396** [comment]
test
to fix your ...,,
https://app.mediafire.com/adsdasdasdasdasdasdasd
password:xxx

**Comment by ABA2396** [comment]
wow

**Comment by Constrat** [comment]
Very cool

**Comment by Constrat** [comment]
https://github.com/don't-click-this-zip.zip

**Comment by SherkeyXD** [comment]
> [!warning]
> 测试样式

**Comment by Constrat** [comment]
> [!Caution]
> 存在未确认的链接，请谨慎访问。
> There are unconfirmed links, please visit with caution.

**Comment by Daydreamer114** [comment]
https://www.bilibili.com/video/BV1oe4y1x7cJ

**Comment by ABA2396** [comment]
pic

**Comment by ABA2396** [comment]
046f539635b55acf8da55c884402a439
github.com 也不安全了

**Comment by Daydreamer114** [comment]
所以来选择gcc吧）

**Comment by ABA2396** [comment]
test
gcc
http

**Comment by ABA2396** [comment]
test
http://baidu.com
gcc changeme

**Comment by ABA2396** [comment]
test
http://baidu.com/

**Comment by status102** [comment]
please install visual studio code select "gcc" plugin

**Comment by ABA2396** [comment]
test
http://baidu.com/
gcc 
changeme

**Comment by SherkeyXD** [comment]
test
changeme

**Comment by Constrat** [comment]
@ABA2396 Since the epidemic stopped. Consider removing?
> [!warning]
存在未确认的链接，请谨慎访问。
There are unconfirmed links, please visit with caution.


We might be able to remove this as well, since I haven't found a single user post this in quite a while. (Github might have finally fixed it)
> [!Caution]
该评论中的链接文件含有病毒，请勿下载。
The file linked in this comment contain a virus. Please do not download it.

**Pull request merged**